deployment/base/scripts/insertDefaults.php

```diff
@@ -179,6 +179,12 @@ protected function doSave(PropelPDO $con)
 
 				$value = file_get_contents($valueFilePath);
 			}
+			
+			if(preg_match('/^#[^#]+$/', $value))
+			{
+			    list($dynamicObjectType, $objectValue) = explode('.', substr($value, 1), 2);
```

Comment:
Dynamic attributes are built of pluginName.value, can't we parse it without knowing the enum-type?